### PR TITLE
New model CLIC_o3_v10

### PR DESCRIPTION
**Pull request by petricm**
BEGINRELEASENOTES
- Introduce new model CLIC_o3_v10
- Unify all encoding strings for ECal, HCal, Yoke with the introduction of GlobalCalorimeterReadoutID

ENDRELEASENOTES

Only Yoke barrel was different from the rest, but for consistency a new variable was introduced.

### CODE REVIEW

**Comment by petricm** [comment]
@andresailer should I change side to -2?

File: lcgeoTests/CMakeLists.txt [hunk 1]
```diff
@@ -110,4 +115,6 @@ ADD_TEST( t_SensThickness_Clic_o3_v8 "${CMAKE_INSTALL_PREFIX}/bin/run_test_${Pac
           ${CMAKE_INSTALL_PREFIX}/bin/TestSensThickness ${CMAKE_SOURCE_DIR}/CLIC/compact/CLIC_o3_v08/CLIC_o3_v08.xml 100 50 )
 ADD_TEST( t_SensThickness_Clic_o3_v9 "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh"
         ${CMAKE_INSTALL_PREFIX}/bin/TestSensThickness ${CMAKE_SOURCE_DIR}/CLIC/compact/CLIC_o3_v09/CLIC_o3_v09.xml 100 50 )
+ADD_TEST( t_SensThickness_Clic_o3_v9 "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh"
```
Comment (andresailer) [inline lcgeoTests/CMakeLists.txt]: This test name needs to be changed

**Comment by andresailer** [comment]
Did you check, if everything is also working with `side:-2`?

**Comment by petricm** [comment]
@webermat testes reco and works with `side:-2`

**Pull request merged**